<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>m_widget</class>
 <widget class="QWidget" name="m_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="guan2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/res/bing2.png);
font: 75 14pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="guan4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/res/bing2.png);
font: 75 14pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="guan3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/res/bing2.png);
font: 75 14pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="guan5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/res/bing2.png);
font: 75 14pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="guan7">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/res/bing2.png);
font: 75 14pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="guan6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/res/bing2.png);
font: 75 14pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="guan8">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/res/bing2.png);
font: 75 14pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="guan9">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/res/bing2.png);
font: 75 14pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="guan1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/res/bing2.png);
font: 75 14pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="chooseLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/res/bing1.png);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; color:#ff5500;&quot;&gt;..选择关卡..&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>700</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Algerian</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>return</string>
   </property>
  </widget>
  <widget class="QLabel" name="ImageLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>801</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文细黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix/start.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>580</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;小提示：后续关卡需要解锁才能进行哟~&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>ImageLabel</zorder>
  <zorder>guan2</zorder>
  <zorder>guan4</zorder>
  <zorder>guan3</zorder>
  <zorder>guan5</zorder>
  <zorder>guan7</zorder>
  <zorder>guan6</zorder>
  <zorder>guan8</zorder>
  <zorder>guan9</zorder>
  <zorder>guan1</zorder>
  <zorder>chooseLabel</zorder>
  <zorder>returnButton</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
